src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
```

Comment:
I think the name of the file should be group.controller and not groupe.controller